app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count = 0
+    without_mask = 0
+    without_vaccine = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            without_vaccine += 1
+        except NotWearingMaskError:
+            without_mask += 1
+        else:
+            count += 1
```

Comment:
If at least one of the friends has issues with the vaccine, you must immediately return "All friends should be vaccinated", no need to create variable for this